test/Feature/HLSLLib/countbits.64.test

```diff
@@ -0,0 +1,98 @@
+#--- source.hlsl
+
+StructuredBuffer<int64_t4> InInt : register(t0);
+StructuredBuffer<uint64_t4> InUInt : register(t1);
+
+RWStructuredBuffer<uint4> OutInt : register(u2);
+RWStructuredBuffer<uint4> OutUInt : register(u3);
+
+[numthreads(1,1,1)]
+void main() {
+  // Int
+  OutInt[0] = countbits(InInt[0]);
+  OutInt[1].xyz = countbits(InInt[0].xyz);
+  OutInt[1].w = countbits(InInt[0].w);
+  OutInt[2].xy = countbits(InInt[0].xy);
+
+  // UInt
+  OutUInt[0] = countbits(InUInt[0]);
+  OutUInt[1].xyz = countbits(InUInt[0].xyz);
+  OutUInt[1].w = countbits(InUInt[0].w);
+  OutUInt[2].xy = countbits(InUInt[0].xy);
+}
+
+//--- pipeline.yaml
+
+---
+Shaders:
+  - Stage: Compute
+    Entry: main
+    DispatchSize: [1, 1, 1]
+Buffers:
+  - Name: InInt
+    Format: Int64
+    Stride: 32
+    Data: [42, 0, 1085102592571150095, -1] # 0x2A, 0x0, 0x0f...0f, 0xff ... ff
+  - Name: InUInt
+    Format: UInt64
+    Stride: 32
+    Data: [42, 0, 1085102592571150095, 1844674407370955161]  # 0x2A, 0x0, 0x0f ... 0f, 0xff ... ff
```

Comment:
The last decimal seems to correspond to 0x01999999999999999 rather than 0xff.......ff. 
I know why the signed integer ones are specified with decimals, but it would be best to specify the unsigned integer ones using hex.